packages/ui-themes/src/sharedThemeTokens/spacing.ts

```diff
@@ -33,7 +34,37 @@ const spacing: Spacing = Object.freeze({
   medium: '1.5rem', // 24px
   large: '2.25rem', // 36px
   xLarge: '3rem', // 48px
-  xxLarge: '3.75rem' // 60px
+  xxLarge: '3.75rem', // 60px
+
+  // new spacing tokens:
+  space0: '0px',
```

Comment:
why are these not in rem units?

Reply:
I agree it should be updated but then `space60` will be a bit misleading when the user is on a zoomed in screen with larger than 16px rems. what do you think the correct solution is here?